nemo_skills/dataset/bfcl_v4/prepare.py

```
@@ -129,9 +141,8 @@ def download_and_process_bfcl_data(repo_url, subfolder_path, output_dir, file_pr
         try:
             # Clone repository with minimal depth
             LOG.info(f"Cloning repository {repo_url} to {temp_dir}")
-            # v1.3 corresponds the release version for BFCL v3
             subprocess.run(
-                ["git", "clone", "-b", "v1.3", "--depth=1", repo_url, temp_dir], check=True, capture_output=True
+                ["git", "clone", "--depth=1", repo_url, temp_dir], check=True, capture_output=True
```

Ruff (0.13.3)

Starting a process with a partial executable path

(S607)